app/models/import.rb

```diff
@@ -31,4 +31,12 @@ def redacted_pdf
 
   def redacted_pdf_url
   end
+
+  def root
+    if parent.is_a?(StandardsImport)
```

Comment:
Alternative implementation in case you want to avoid an `is_a?` check:

```ruby
class Import < ApplicationRecord
  def import_root
    parent.import_root
  end
end

class StandardsImport < ApplicationRecord
  def import_root
    self
  end
end
```

Reply:
Oh yes, that's nicer, thanks! 605c7d581722e48a930bd4c239b397b2212e5e05